### PR TITLE
Set relations helper when creating event tile context menu

### PR DESCRIPTION
**Pull request by Johennes**
Fixes https://github.com/vector-im/element-web/issues/22018

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Set relations helper when creating event tile context menu ([\#9253](https://github.com/matrix-org/matrix-react-sdk/pull/9253)). Fixes vector-im/element-web#22018.

### CODE REVIEW

**Review by turt2live — COMMENTED** [review]
this seems fine code-wise, but a regression test at a minimum would be appreciated.

**Comment by Johennes** [comment]
> this seems fine code-wise, but a regression test at a minimum would be appreciated.

I couldn't find a better way of doing this other than hooking into the existing e2e test. Let me know if that is ok.

File: cypress/e2e/polls/polls.spec.ts [hunk 1]
```diff
@@ -157,6 +157,91 @@ describe("Polls", () => {
         });
     });
 
+    it("Polls without votes can be edited from context menu", () => {
```
Comment (turt2live) [inline cypress/e2e/polls/polls.spec.ts:160]: fwiw, test naming is typically in the following form:

```js
describe("TheTheThingBeingTested", () => {
  it("should have a particular behaviour", () => { ... });
});
```

for example, this test would be something like:

```js
describe("Polls", () => {
  it("should be editable if they have no votes", () => { ... });
});
```

in the grand scheme of things it doesn't really matter - just spreading the word for eventual consistency :)

**Review by turt2live — APPROVED** [review]
lgtm - thanks :)

File: cypress/e2e/polls/polls.spec.ts (hunk 1 above)
Comment (Johennes) [reply cypress/e2e/polls/polls.spec.ts]: Actually that was my understanding, too. I just followed the style used in the other test methods in this file. I've switched all of them over to the idiomatic format now.

**Comment by Johennes** [comment]
@turt2live I think this is good to merge now.

**Review by turt2live — APPROVED** [review]

**Comment by Johennes** [comment]
@turt2live could you merge this one for me please?

**Comment by turt2live** [comment]
Sure, also I'm going to add you to a useful team for this.

**Pull request merged**